surah/src/main/res/drawable/aya_circle_shape.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item>
+        <shape android:shape="rectangle">
+            <gradient
+                android:centerColor="#ffe2f1f6"
+                android:endColor="#ffe2f1f6"
+                android:gradientRadius="56dp"
+                android:startColor="#802BB5A8"
+                android:type="radial" />
+            <padding
+                android:bottom="36dp"
+                android:left="36dp"
+                android:right="36dp"
+                android:top="36dp" />
```

Comment:
dimensions and colors could be referenced from resources(dimens/colors) and make future changes easier.

Reply:
Sorry this file was not supposed to be here anymore, thanks for reminding